### DISCUSSION TITLE
Spook Boo! service - not very clear why one would need this

### DISCUSSION THREAD

**Discussion by vingerha** [frenck/spook#621] [Documentation]
I read that it stops all (I guess) running automations with an error. Can you possibly provide a use-case?

**Comment by frenck** [comment]
It doesn't stop all automations with an error. Where did you read that?

> Can you possibly provide a use-case?

There is none, also written in the documentation.

../Frenck

**Reply by vingerha** [reply]
misread 'all' and sadly no use case :)
